src/commands/Command.java

```diff
@@ -0,0 +1,7 @@
+package commands;
+
+/**
+ * Created by TANIA BARRIONEVO on 18/05/2016.
+ */
+public class Command {
```

Comment:
If this is going to be worked later, please add a TODO comments or something that gives that idea